### ISSUE TITLE
NTR: Small intestine biopsy: increased intraepithelial lymphocytes (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8663]
New term request Small intestine biopsy: increased intraepithelial lymphocytes (nadea3_200428185729):   UMLS: 3854615

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000251
- label: Small intestinal intraepithelial lymphocytosis
- parents: [Abnormal small intestinal mucosa morphology (HP:0025129)](https://hpo.jax.org/app/browse/term/HP:0025129)
- pmid: PMID:8669535
- def: The count of lymphocytes located in the epithelial layer of the mucosa of the small intestine is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8663
- synonyms:
